src/20_Components/amp-bind.html

```diff
@@ -101,27 +101,29 @@
    set when a state change is triggered (see below).
 
 -->
-<p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+<div class="p1">
+  <p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+</div>
 
 <!-- ## Binding Styles -->
 <!--
   Styling can be changed by applying css classes. It will override all the element's style classes. `allAnimals` is the `id` of the &lt;amp-state&gt; component above.
 -->
-
-<p [class]="allAnimals[currentAnimal].style" class="greenBackground">Each animal has a different background color.</p>
-
+<div class="p1">
+  <p [class]="allAnimals[currentAnimal].style" class="greenBackground">Each animal has a different background color.</p>
+</div>
 <!-- ## Binding Attribute Values -->
 <!--
 Various AMP components support binding attribute values with amp-bind. For example, `amp-img` URLs can be changed.
 -->
-  <amp-img src="/img/Border_Collie.jpg" width="300" height="200"
+  <amp-img src="/img/Border_Collie.jpg" class="m1" width="300" height="200"
       [src]="allAnimals[currentAnimal].imageUrl">
   </amp-img>
 
   <!--
     The src URL for videos embedded with `amp-video` can be changed. At the moment, among all the AMP video components, only amp-video supports amp-bind.
   -->
-  <amp-video src="/video/dog-video.mp4" [src]="allAnimals[currentAnimal].videoUrl" width="300" height="200" autoplay controls="true"></amp-video>
+  <amp-video src="/video/dog-video.mp4" class="m1" [src]="allAnimals[currentAnimal].videoUrl" width="300" height="200" autoplay controls="true"></amp-video>
```

Comment:
same as above, I'd prefer to keep the video full-bleed but use a responsive layout